README.rst

```diff
@@ -24,3 +24,16 @@ If you are interested in contributing or extending the project, take a look at t
 .. |Requirements Status| image:: https://requires.io/github/openoereb/pyramid_oereb/requirements.svg?branch=master
    :target: https://requires.io/github/openoereb/pyramid_oereb/requirements/?branch=master
    :alt: Requirements Status
+
+tl;nr - Local testing
```

Comment:
Is this a typo? (The t1;nr part)

Reply:
=> https://en.wiktionary.org/wiki/TL;NR

But perhaps that's not so common ? I have changed the title.